### ISSUE TITLE
k8s explorer panel hides after closing and reopening the project

### ISSUE THREAD

**Issue by etanshaul** [GoogleCloudPlatform/cloud-code-intellij#2564]
This is rather strange:

1) have the explorer tab open
2) close the project (so that you go back to the welcome screen)
3) minimize the explorer tab (by clicking on the tab)

Expected:
The panel just minimizes

Actual:
The panel minimizes AND the tab disappears entirely. Have to go back to the tools menu to reopen it.

**Comment by ivanporty** [comment]
I don't think this is a bug, this looks like a behavior of "non-lockable panel" when you open it first. I'll just make it "lockable" whatever that means.

**Comment by etanshaul** [comment]
lol. cool. That's odd. makes me think its a platform bug.

**Comment by etanshaul** [comment]
possibly related:
https://youtrack.jetbrains.com/issue/IDEA-153491
https://youtrack.jetbrains.com/issue/IDEA-150455

**Comment by etanshaul** [comment]
@ZariaHoward wanna see if you can figure out how to make the panel "lockable" (see @ivanporty's above comment)? This is a super strange bug which we should avoid on our end if we can.
